### DISCUSSION TITLE
mac版本Stellarium的星历表（DE430, DE431, DE440, DE441）如何设置？

### DISCUSSION THREAD

**Discussion by jesen777** [Stellarium/stellarium#3320] [Q&A] [answered]
mac版本Stellarium的星历表（DE430, DE431, DE440, DE441）如何设置？

**Comment by jesen777** [comment]
在Finder里搜索config.ini，一般会搜到好几个相同的文件，
右键在“显示简介”里的位置确定是属于Stellarium的config.ini，
然后用文本编辑打开，用花键+F搜索de430，可以找到两个目标。
第一个是在[astro]下，路径直接给你留好了，在引号里加入就是了：
（我是直接把下载好的四个星历表放进了天文馆程序包里）
/Applications/Stellarium.app/linux_p1550p2650.430
/Applications/Stellarium.app/lnxm13000p17000.431
类似的并没有留出440和441的路径，直接复制然后添加路径上去就OK了。
/Applications/Stellarium.app/linux_p1550p2650.440
/Applications/Stellarium.app/linux_m13000p17000.441
第二个是赋值是false，这个不用管，相同的照搬复制两个440和441类目上去就行了，
然后等你打开软件发现灰色的星历表都可以用了，等你都打上勾关联了，第二条会自动赋值true的。
大功至此告成!

**Comment by gzotti** [comment, marked as answer]
Machine translations:
OP: How to set the ephemeris (DE430, DE431, DE440, DE441) of the mac version Stellarium?

Follow-up:
Search for config.ini in Finder, usually you will find several identical files, The position of the right button in the "Display Profile" is sure to belong to Stellarium's config.ini, Then open it with text editor, search for de430 with spline + F, you can find two targets. The first one is under [astro], the path is reserved for you directly, just add it in the quotation marks: (I directly put the downloaded four ephemeris into the planetarium package) /Applications/Stellarium.app/linux_p1550p2650.430 /Applications/Stellarium.app/lnxm13000p17000.431 Similarly, there is no path for 440 and 441, just copy it and add the path to it and it will be OK. /Applications/Stellarium.app/linux_p1550p2650.440 /Applications/Stellarium.app/linux_m13000p17000.441 The second is that the assignment is false, so don’t worry about it, just copy the two 440 and 441 categories in the same way. Then when you open the software, you will find that the gray ephemeris is available, and when you have ticked and connected, the second item will be automatically assigned true. That's it!

Correction: 
The User Guide provides instructions in section 5.5.3. Using a folder "ephem" in your user data directory does not even require editing config.ini. You should avoid copying any file into the program directory unless you know what you are doing.
